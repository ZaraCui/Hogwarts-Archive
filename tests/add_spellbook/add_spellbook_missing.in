ADD COLLECTION tests/data/books_small.csv
ADD SPELLBOOK tests/data/books_partial.csv 9999
EXIT

