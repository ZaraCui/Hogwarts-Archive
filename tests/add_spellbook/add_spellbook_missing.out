user: 12 spellbooks successfully added.

user: No such spellbook in file.

user: Ending Archive process.
